### ISSUE TITLE
[Quest] The Exorcism of Colonel Jules

### ISSUE THREAD

**Issue by PoisonStream** [WoWManiaUK/Redemption#1428]
This quest in the Outlands is bugged. My friend tried to do it and it spawned too many enemies. Me and her tried to do it together, it spawned a lot of enemies but we killed all of them at the start... then the colonel flew out of the Inn through the wall and was outside spawning the monsters... Honor Hold was full of them.

**Comment by tvshadowz** [comment]
Hi there PoisonStream! 

In future could you follow these guidelines http://www.wow-mania.com/forums/topic/71 on how a bug should be report because it helps us testers alot!

Just in future make sure you link quests and screenshots if you have them - thanks! :)

**Comment by tvshadowz** [comment]
Issue:

During the scenario NPC Anchorite Barada can be spoken to and the excorcism can be started multiple times. I believe the user PoisonStream may have encountered this issue with their friend due to starting the phase multiple times. 

I did not encounter the issue with Colonel Jules flying out of the inn.

**Comment by Katherwind** [comment]
Just wanted to let you know that this quest was still bugged as of Friday evening. My husband started the quest. I assisted him in killing all the mobs in the room that had spawned and the exorcism was complete.  However, Colonel Jules was nowhere to be found to talk to complete the quest.  We went outside the inn, mounted our flying mounts and flew around the inn.  The area in the sky around the inn was filled with spawned mobs. 

We went back inside the inn and up to the room .  Colonel Jules had respawned but we could not talk to him to complete the quest.
